docs/Analyzer Configuration.md

```diff
@@ -264,6 +264,7 @@ Option Name: `excluded_symbol_names`
 Configurable Rules:
 [CA1001](https://docs.microsoft.com/visualstudio/code-quality/ca1001),
 [CA1062](https://docs.microsoft.com/visualstudio/code-quality/ca1062),
+[CA1068](https://docs.microsoft.com/visualstudio/code-quality/ca1068),
```

Comment:
The docs needs to be updated to mention the configurability of the rule done in this PR, in addition to the recent configurability change done with https://github.com/dotnet/roslyn-analyzers/issues/2851 for the same rule.

Reply:
Thanks @Youssef1313 - agreed.

Reply:
https://github.com/dotnet/docs/pull/21853